docs/GSSOC_CONTRIBUTOR_GUIDE.md

```
@@ -12,79 +12,126 @@ GirlScript Summer of Code (GSSoC) is a 3-month open-source program conducted by
+Issues are manually categorized using difficulty and type labels to help contributors choose tasks matching their experience level.
 
-New issues are **automatically triaged by our AI bot** which applies difficulty and type labels. You can filter by these labels to find issues that match your skill level.
+---
 
-### Step 2 — Request Assignment
+## Step 2 — Request Assignment
 
 Comment on the issue with:
 
-```bash
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)